src/main/java/org/apache/sling/discovery/impl/standalone/Config.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.sling.discovery.impl.standalone;
+
+import org.osgi.service.metatype.annotations.AttributeDefinition;
+import org.osgi.service.metatype.annotations.ObjectClassDefinition;
+
+/**
+ * Configuration interface for the standalone discovery service.
+ */
+@ObjectClassDefinition(
+    name = "Apache Sling Discovery Standalone Configuration",
+    description = "Configuration for the standalone discovery service implementation"
+)
+public @interface Config {
+
+    @AttributeDefinition(
+        name = "Always Offline",
+        description = "When enabled, this instance will always report itself as offline/inactive in the topology"
```

Comment:
I would slightly rephrase this. the discovery-standalone is inherently running standalone. i.e. it is not taking part in any topology. So the "in the topology" part seems a bit misleading. Maybe rather focus on the users of discovery: it will never report a valid topology and never send a TOPOLOGY_INIT. Something in that direction